geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/cluster/RedisMemberInfo.java

```diff
@@ -26,7 +27,7 @@
 import org.apache.geode.internal.serialization.KnownVersion;
 import org.apache.geode.internal.serialization.SerializationContext;
 
-public class RedisMemberInfo implements DataSerializableFixedID {
+public class RedisMemberInfo implements DataSerializableFixedID, Serializable {
```

Comment:
The `RedisMemberInfo` is returned as part of a DUnit call. Since that doesn't use any custom Geode serialization, it needs to support java's generic serialization.